### PR TITLE
Update readme with draft project description

### PR DESCRIPTION
**Pull request by s-espinosa**
Add draft readme with draft project description.

* Outstanding questions:
    * Do we want students to take cities out of the stations table? If not, recommend removing/adjusting note in Iteration 1 regarding normalization.
    * Currently this includes pagination for trips. Is this description sufficient? Do we also want to allow users to see a list of links so that they can skip ahead to pages down the line?
    * If we keep pagination, do we need to be more explicit in how they will feed page data into their controller or do we think they can figure it out?
    * Do we want to include the data in this repository or have them upload it elsewhere?
* Two Rubric's (previous version and current draft). Left the previous version in currently to use as reference.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,324 @@
...
+
+In this project you'll use Ruby, Sinatra, and Activerecord to build a site that analyzes bike share usage in San Francisco.
+
+This application will collect data in two ways: 1) through web forms allowing users to enter trip/station/weather information, and 2) by consuming CSV files with historical information for stations, trips, and weather. In addition to creating and storing this information for viewing at a later date, this application will provide a number of user dashboards with higher level analysis of trends in bike share usage.
+
+At a high level, this project will develop along the following path:
+
+* Iteration 1: Allow users to save and view information related to bike share stations.
+* Iteration 2: Create a seed file to consume information on the seventy existing bike share stations in San Francisco.
+* Iteration 3: Create a station dashboard with some analysis of station information (split by city, date installed, number of of bicycle docks)
```
Comment (case-eee) [inline README.md]: I'd remove the word "some" here

File: README.md [hunk 2]
```diff
@@ -1 +1,324 @@
...
+### ActiveRecord
+
+* Use ActiveRecord migrations to create a normalized database
+* Use intermediate ActiveRecord queries to calculate and report on information in the database
+* Utilize ActiveRecord methods and relationships to efficiently query the database
+
+### User Experience and Conventions
+
+* Use Sinatra and ERB templates to render views to create, read, update, and delete resources using restful routes and appropriate HTTP verbs
+* Use Sinatra and ERB templates to display a dashboard not related to a specific resource saved in the database
```
Comment (case-eee) [inline README.md:32]: Isn't the dashboard related to the data in the database?

File: README.md [hunk 3]
```diff
@@ -1 +1,324 @@
...
+
+### Working Collaboratively
+
+* (unevaluated) Use Git and GitHub to work collaboratively, develop in smaller groups, and resolve merge conflicts
+
+## Getting Started
+
+### Define the Relationship with Your Group
+
+DTR with youg group [here](https://gist.github.com/case-eee/38e212c799c563f58766128b5057858a). ONe group member should send a link of the forked gist to your anchor as soon as it's complete.
```
Comment (case-eee) [inline README.md]: Capitalization here

File: README.md [hunk 4]
```diff
@@ -1 +1,324 @@
...
+
+### Clone the Project
+
+1. One team member forks the skeleton repository [here](TBD).
```
Comment (case-eee) [inline README.md]: They'll fork this repository

File: README.md [hunk 5]
```diff
@@ -1 +1,324 @@
...
+* city
+* installation date
+
+Be sure to normalize the table(s) that you build to hold this information, and to use restful routes. At the end of this iteration, you should be able to view an index of all stations, view a page for a single station, create a station, edit a station, and delete a station from either the index or the show pages.
+
+### Iteration 2
+
+Download the dataset available [here](https://www.kaggle.com/benhamner/sf-bay-area-bike-share). This will include all of the CSV files that you will be using in this project. Since these files are large, we have set up your project directory structure and `.gitignore` so that when you add these files to the `/db/csv/` directory they will not be pushed up to GitHub. However, this also means that each of the members of your team will need to download these files independently.
+
+Create a seed file in your `/db` directory and add the `station.csv` file to your `/db/csv/` directory. When you run `ruby db/seed.rb` your development database should be populated with the information from the `station.csv` file. Your index should include a total of seventy stations.
```
Comment (case-eee) [inline README.md]: I believe I created an empty seed file already so they just need to add to it

**Review by case-eee — COMMENTED** [review]
I looked through everything but the rubric - mainly just commented on small things, this is great 👍🏼 I'll look through the rubric in a bit!

File: README.md (hunk 2 above)
Comment (s-espinosa) [reply README.md:32]: It is, but not a resource? The distinction that I'm trying to draw is between a dashboard, which would use a custom route and something that would use a restful route. The goal was to hammer home the idea that you introduced this week that there will be times that it makes sense to not follow restful conventions. Is there a better way to say this here?

**Comment by s-espinosa** [comment]
Thanks! Made changes based on these comments. Will update further based on your comments on the rubric. Had one question for you regarding `resources`. Might be that it's just not worth drawing that distinction. Let me know if you think there's a better way to capture that or if it's better to just drop it.

File: README.md (hunk 2 above)
Comment (case-eee) [reply README.md:32]: Gotcha. This makes sense now. The way you have it worded here is fine. Let me think on a way to make it even clearer

**Comment by case-eee** [comment]
I like having them extract cities. I touched on that a bit today.

As far as pagination goes, I think we can keep it. How hard do we think it will be for them to implement? Do we want them to do it on their own or give them support?

**Comment by s-espinosa** [comment]
Re: extracting cities - Sounds good! Think that means we can leave the description as is.
Re: pagination - I'm currently leaning towards letting them do it on their own? I think we'll get some different implementations (maybe some a little off the wall), but the upside is that it will require some problem solving (which I like). I'd imagine they'd get to it sometime over the weekend, so it might be something we can clean up on Monday. Maybe we check with Ellen Mary?

**Pull request merged**